src/test/rgw/test_rgw_string.cc

```diff
@@ -12,6 +12,7 @@
  *
  */
 
+#include <errno.h>
```

Comment:
sorry, i don't follow..

if you'd have to include `errno.h` in every file that included `rgw_string.h`, surely `rgw_string.h` should just include `errno.h` instead

or am i missing the actual error here?